packages/ui-options/src/Options/README.md

```diff
@@ -130,7 +130,11 @@ class Example extends React.Component {
         width="300px"
         shadow="above"
       >
-        <Options onKeyDown={this.handleKeyDown} tabIndex="0">
+        <Options
+          onKeyDown={this.handleKeyDown}
+          onMouseOut={(e) => this.setState({ highlighted: -1 })}
+          tabIndex="0"
+        >
```

Comment:
Just added this logic to the docs examples because the hover state was stuck if you hovered over it once.